processing/src/main/java/io/druid/segment/serde/DictionaryEncodedColumnPartSerde.java

```diff
@@ -364,4 +386,9 @@ public void read(ByteBuffer buffer, ColumnBuilder builder, ColumnConfig columnCo
       }
     };
   }
+
+  private static boolean mustWriteFlags(final int flags)
+  {
+    return flags != NO_FLAGS && flags != Feature.MULTI_VALUE.getMask();
```

Comment:
Multi value V3 mask ?

Reply:
MUTLI_VALUE_V3 must be written so that's why it's not listed here.

Reply:
please add a comment to code.

Reply:
Added a comment.